packages/react/src/popover/root/usePopoverRoot.ts

```diff
@@ -103,11 +104,11 @@ export function usePopoverRoot(params: usePopoverRoot.Parameters): usePopoverRoo
       }
 
       if (isHover) {
-        // Prevent impatient clicks from unexpectedly closing the popover.
-        setClickEnabled(false);
-        clearTimeout(clickEnabledTimeoutRef.current);
-        clickEnabledTimeoutRef.current = window.setTimeout(() => {
-          setClickEnabled(true);
+        // Only allow "patient" clicks to close the popover if it's open.
+        // If they clicked within 500ms of the popover opening, keep it open.
+        clearTimeout(stickIfOpenTimeoutRef.current);
+        stickIfOpenTimeoutRef.current = window.setTimeout(() => {
+          setStickIfOpen(false);
```

Comment:
While this new implementation makes more sense logically, it makes infotips like the ones on the docs work less well. Ignoring clicks is favorable there because it prevents multiple ones from being open simultaneously. The only way to get that to happen with the master version is to click it after closing/multiple times. cc: @vladmoroz

Reply:
Using `Popover.Backdrop` (or adding modality in general, at least for clicks) would actually solve this, since it would block pointer-events of everything underneath until they intentionally dismiss with outside press